lib/fluent/supervisor.rb

```diff
@@ -1196,6 +1205,28 @@ def build_system_config(conf)
       system_config
     end
 
+    def build_additional_configuration
+      if @system_config.config_include_dir&.empty?
+        $log.info :supervisor, 'default configuration include directory was disabled'
+        return
+      end
+      begin
+        Find.find(@system_config.config_include_dir) do |path|
+          next if File.directory?(path)
+          next unless [".conf", ".yaml", ".yml"].include?(File.extname(path))
+          # NOTE: both types of normal config (.conf) and YAML will be loaded.
+          # Thus, it does not care whether @config_path is .conf or .yml.
+          $log.info :supervisor, 'loading additional configuration file', path: path
+          yield Fluent::Config.build(config_path: path,
+                                         encoding: @conf_encoding,
+                                         use_v1_config: @use_v1_config,
+                                         type: :guess)
+        end
+      rescue Errno::ENOENT
+        $log.warn :supervisor, 'inaccessible include directory was specified', path: @system_config.config_include_dir
```

Comment:
```suggestion
        $log.info :supervisor, '"config_include_dir" does not exist. Does not load additional configuration files.', path: @system_config.config_include_dir
```

There are 2 points.

* How about changing the log level to `info`.
* It would be better to make the message specific so that the user knows what the warning is about.

In the current implementation, the following warning occurs when I run raw Fluentd by default.

```
2025-04-17 18:26:42 +0900 [warn]: inaccessible include directory was specified
 path="/etc/fluent/conf.d"
```

This would be confusing because a user specifies nothing, but it warns about a path the user may not know.

If we use `warn` level here, it would be better to limit it to cases where the user has explicitly configured it.
Mayby we don't have to go that far because it can be assumed that there are few cases where a user actually configures this path explicitly, other than to disable this feature in the package.